portal_multicurrency_totals/demo/res_partner_demo.xml

```diff
@@ -27,7 +17,7 @@
             eval="[
                 Command.create(
                     {
-                        'user_id': obj(ref('base.res_partner_3')).user_ids.id,
+                        'user_id': obj(ref('base.res_partner_3')).user_ids.ids[0],
```

Comment:
Why this?

The partner should have only one user, shouldn't it.?

Reply:
The partner could be linked to multiple users. However, now .id doesn’t return the first ID as it did before. That’s why .ids with an index has to be used.

Reply:
It's the same behavior:
- `.id` returns an integer
- `.ids` returns a list

I think it's not necessary. But if you prefer to assure that, I suggest `user_ids[0].id` instead

Even though it's technically possible, a user should only be linked to a single partner. The field is a o2m because the inverse field is a m2o.

Reply:
Suggestion applied.